### PR TITLE
ARSN-155 - export utils for cloudserver

### PR DESCRIPTION
**Pull request by jbertran**

### CODE REVIEW

**Comment by jbertran** [comment]
Woops

**Comment by miniscruff** [comment]
I have https://scality.atlassian.net/browse/ARSN-150, should be up today I was testing it locally.

File: index.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export const versioning = {
 export const network = {
     http: {
         server: require('./lib/network/http/server'),
+	utils: require('./lib/network/http/utils'),
```
Comment (miniscruff) [inline index.ts]: spacing is wrong
```suggestion
	    utils: require('./lib/network/http/utils'),
```

Comment (jbertran) [reply index.ts]: :man_facepalming:

**Review by miniscruff — APPROVED** [review]

**Review by ghivert — APPROVED** [review]

**Pull request merged**